tsl/src/nodes/vector_agg/exec.h

```diff
@@ -3,7 +3,14 @@
  * Please see the included NOTICE for copyright information and
  * LICENSE-TIMESCALE for a copy of the license.
  */
-#pragma once
```

Comment:
Is the removal of the `#pragma` intended?

Reply:
Returned it back.